Lib/test/test_typing.py

```diff
@@ -4091,6 +4091,25 @@ def method(self) -> None: ...
         self.assertIsInstance(Foo(), ProtocolWithMixedMembers)
         self.assertNotIsInstance(42, ProtocolWithMixedMembers)
 
+    def test_protocol_issubclass_error_message(self):
+        class Vec2D(Protocol):
+            x: float
+            y: float
+
+            def square_norm(self) -> float:
+                return self.x ** 2 + self.y ** 2
+
+        assert Vec2D.__protocol_attrs__ == {'x', 'y', 'square_norm'}
```

Comment:
Never use bare `assert`s in tests run with unittest

```suggestion
        self.assertEqual(Vec2D.__protocol_attrs__, {'x', 'y', 'square_norm'})
```